hilfe/handbuch/datenexport/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: layouts/hilfe/single.njk
+title: Datenexport
+icon: download
+---
+
+Unter Datenexport können Sie Daten von Pentacode zusammengefasst herunterladen und in andere Software übertragen. In den verschiedenen Gebieten von Pentacode finden Sie öfters das {%icon "download"%}**-Symbol**, dieses verweist Sie stets in den hier beschriebenen Reiter.
+
+## Lohnbuchhaltung
+
+Hier finden Sie alle Exportoptionen, die der Lohnvorbereitung und -dokumentation dienen.  
+
+### Lohnbericht Übersicht
+
+In der Lohnbericht-Übersicht, sind die Arbeitsstunden und -tage, sowie Zuschlagsstunden, Mitarbeiteressen und das aktuelle Gehalt in einer tabellarischen Ansicht zusammengefasst. Dieser Ausdruck ist vor allem für die Übermittlung an Ihr Lohnbüro gut geeignet und dient als Alternative zum direkten Export/Import in Ihr Lohnbuchhaltungs-Programm.
+Wählen Sie dazu unter **Lohnbericht (Übersicht)** den gewünschten Zeitraum aus. Unter **Mitarbeiter** können Sie nun auswählen für wen Sie die Lohnberichtübersicht erstellen wollen - dabei können Sie ganze Abteilungen oder einzelne Mitarbeiter auswählen. Wählen Sie eine **Abteilung und einen Mitarbeiter, werden Ihnen nur die Lohndaten der Schichten des Mitarbeiters in dieser Abteilung angezeigt**.
+Geben Sie eine Email Adresse in das Feld über {%button label = "**Export starten**" %} ein, um die Lohnberichtsübersicht direkt zu verschicken. 
+
+{% figure caption="Hier Exportieren wir die Lohnberichte aller Manager - wie das **Symbol** zeigt ist davon nur ein Mitarbeiter betroffen" %}
+<img src="lohnbericht_übersicht.webp" />
+{% endfigure %}
+
+-- Wie Icons ins Captions einfügen?
+
+
+### Lohnabrechnung Detailliert
+
+Die detaillierte Lohnübersicht fasst **die Inhalte des Lohnberichts mit allen Arbeitszeiten zusammen!** So haben Sie in einem Dokument alle relevanten Lohndaten und können gleichzeitig noch überprüfen, durch welche Arbeitseinsätze sich diese zusammensetzen. 
+Wählen Sie dazu **Lohnbericht (Detailliert)** aus und geben wieder den gewünschten Zeitraum sowie die Mitarbeiter / Abteilungen ein, für den Sie die detaillierte Lohnübersicht erstellen möchten. Auch hier haben Sie wieder die Möglichkeit die Daten **direkt per Email zu verschicken**. 
```

Comment:
Der Lohnbericht Detailliert enthält zusätzlich:

- Eine List der Abwesenheiten mit Beginn, Ende und Anzahl der Fehltage
- Vertragsveränderungen bzw. Vertragsende, falls in dem Zeitraum vorhanden
- Veränderungen des Arbeitszeitkontos